src/tables/GPOS.js

```diff
@@ -185,7 +185,7 @@ let GPOSLookup = new r.VersionedStruct('lookupType', {
   9: { // Extension Positioning
     posFormat:   r.uint16,
     lookupType:  r.uint16,   // cannot also be 9
-    extension:   new r.Pointer(r.uint32, GPOSLookup)
```

Comment:
Hmm, this doesn't look like an eval/new Function removal.

Reply:
@Pomax: true. But it's a fix for modern browser that honor 'let'. The varible GPOSLookup is not defined at that point at time anyway. If converted to 'var' it is undefined.

Reply:
I would strongly recommend filing a separate PR for that. As a PR to fix a CSP problem, this line (and the one for GSUB) are not part of the issue being fixed, and deserve their own issue, with their own PR.